estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,40 @@
+from odoo import fields, models, api
+from odoo.tools import date_utils as du
+from datetime import date
+
+
+class EstatePropertyOffer(models.Model):
+    _name = "estate.property.offer"
+    _description = "Offers made on a listing"
+    price = fields.Float()
+    status = fields.Selection(
+        string="Status",
+        selection=[
+            ("accepted", "Accepted"),
+            ("refused", "Refused"),
+        ],
+        copy=False,
+    )
+    partner_id = fields.Many2one("res.partner", string="Buyer", required=True)
+    property_id = fields.Many2one("estate.property", string="Property", required=True)
+    validity = fields.Integer(default=7, string="Validity (days)")
+    date_deadline = fields.Date(
+        compute="_compute_date_deadline",
+        inverse="_inverse_date_deadline",
+        string="Deadline",
+    )
+
+    @api.depends("validity")
+    def _compute_date_deadline(self):
+        for record in self:
```

Comment:
nitpick: use descriptive names for iterator variables please ^^
something like
```suggestion
        for offer in self:
```